vars/commonPipelineEnvironment.groovy

```diff
@@ -17,6 +17,8 @@ class commonPipelineEnvironment implements Serializable {
     String gitHttpsUrl
     String gitBranch
 
+    String xsDeploymentId
```

Comment:
In case of a blue-green deployment the `xsDeploy` step gets called in two different modes (first for the deployment and the second time for resuming or aborting it). Hence we need this here for remembering the deployment id.

Other possible approach. Have two steps: one for the xsdeploy and anotherone (or two) for resuming/aborting it. There are pros/cons for both, having one or several steps. Basically both options are valid for me. In case someone prefers to have smaller steps I can change that.